arch/risc-v/src/mpfs/mpfs_opensbi.c

```diff
@@ -232,15 +232,9 @@ static const struct sbi_platform platform =
 
 /* This must go into l2_scratchpad region, starting at 0x0a000000. */
 
-static sbi_scratch_holder_t g_scratches[MPFS_MAX_NUM_HARTS] \
+sbi_scratch_holder_t g_scratches[MPFS_MAX_NUM_HARTS] \
                __attribute__((section(".l2_scratchpad")));
 
-/* These stacks are used in the mpfs_opensbi_utils.S */
-
-uint8_t g_hart_stacks[SBI_PLATFORM_DEFAULT_HART_STACK_SIZE * \
-                      MPFS_HART_COUNT] \
```

Comment:
Maybe this is no londer needed (line 67):
#if SBI_PLATFORM_DEFAULT_HART_STACK_SIZE != 8192
#  error "Fix define in file mpfs_opensbi_utils.S"
#endif

Reply:
Yes it can be removed!